dashbuilder-client/dashbuilder-navigation-client/src/main/java/org/dashbuilder/client/navigation/widget/HasTargetDiv.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.dashbuilder.client.navigation.widget;
+
+/**
+ * Interface implemented for those {@link NavWidget} implementations that accept a target div as the location to
+ * display the content of the nav items clicked by the user.
+ */
+public interface HasTargetDiv {
```

Comment:
HasTargetDiv means the component can be configured with a target div component reference. The item's display is sent to that target div instead of being handled by the own component.